src/patterns/definitions.owl

```diff
@@ -7,8 +7,142 @@ Prefix(rdfs:=<http://www.w3.org/2000/01/rdf-schema#>)
+
+# Class: <http://purl.obolibrary.org/obo/CL_4033004> (smooth muscle cell of taenia coli)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "PMID:10715339"^^xsd:string) Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "PMID:22320805"^^xsd:string) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/CL_4033004> "A(n) smooth muscle cell that is part of a(n) taenia coli."^^xsd:string)
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> <http://purl.obolibrary.org/obo/CL_4033004> <https://orcid.org/0000-0001-6677-8489>)
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/CL_4033004> "smooth muscle cell of taenia coli"^^xsd:string)
+EquivalentClasses(<http://purl.obolibrary.org/obo/CL_4033004> ObjectIntersectionOf(<http://purl.obolibrary.org/obo/CL_0000192> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000050> <http://purl.obolibrary.org/obo/UBERON_0012419>)))
+
+# Class: <http://purl.obolibrary.org/obo/CL_4033005> (serous secreting cell of bronchus submucosal gland)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "PMID:5487122"^^xsd:string) Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "PMID:9651178"^^xsd:string) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/CL_4033005> "A(n) serous secreting cell that is part of a(n) bronchus submucosal gland."^^xsd:string)
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> <http://purl.obolibrary.org/obo/CL_4033005> <https://orcid.org/0000-0001-6677-8489>)
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/CL_4033005> "serous secreting cell of bronchus submucosal gland"^^xsd:string)
+EquivalentClasses(<http://purl.obolibrary.org/obo/CL_4033005> ObjectIntersectionOf(<http://purl.obolibrary.org/obo/CL_0000313> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000050> <http://purl.obolibrary.org/obo/UBERON_8410043>)))
+
+# Class: <http://purl.obolibrary.org/obo/CL_4033006> (endothelial cell of lymphatic vessel of efferent lymphatic vessel)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "PMID:32094869"^^xsd:string) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/CL_4033006> "A(n) endothelial cell of lymphatic vessel that is part of a(n) efferent lymphatic vessel."^^xsd:string)
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> <http://purl.obolibrary.org/obo/CL_4033006> <https://orcid.org/0000-0001-6677-8489>)
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/CL_4033006> "endothelial cell of lymphatic vessel of efferent lymphatic vessel"^^xsd:string)
```

Comment:
Is there a particular reason this is not labelled simply as 'endothelial cell of efferent lymphatic vessel'?

Reply:
done